app/test_main.py

```diff
@@ -1,3 +1,17 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_zero_when_ages_less_15():
+    assert get_human_age(0, 14) == [0, 0]
+
+
+def test_should_return_1_when_ages_between_15_and_23():
+    assert get_human_age(15, 23) == [1, 1]
+
+
+def test_should_add_1_year_every_4_pets_years_when_ages_24_and_more():
+    assert get_human_age(100, 28) == [21, 2]
```

Comment:
it would be better to check the boundary cases like the 3-rd and 2-nd years of cat_and_dog_years life like you did for a 1-st year, cower from both sides - (14=> 0, 15 => 1)